src/App.js

```diff
@@ -1,7 +1,29 @@
+import {
+  Route,
+  Switch,
+  useLocation,
+} from 'react-router-dom';
+import NavBar from './components/nav/NavBar';
```

Comment:
- [ ] Please add a CreateNewBook component also. it should render a form with inputs for the title and the author of the book.
- [ ] It should also include a button to submit, which would also just be a placeholder for now.

Reply:
Done!